support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -27,8 +27,9 @@ export const pageUrlRegexes = {
 			paperLandingPage: /^\/uk\/subscribe\/paper?(\?.*)?$/,
 			weeklyLandingPage: /\/subscribe\/weekly\/checkout?(\?.*)?$/,
 		},
+		// Includes landing, original & generic checkout/thankyou pages
 		subsWeeklyPages:
-			'(/??/subscribe(\\?.*)?$|/??/subscribe/weekly(\\/checkout)?(\\?.*)?$)',
+			'(/subscribe/weekly/checkout$)(/.*)?$|(?:/(uk|us|ca|eu|nz|int))(?:/(subscribe/weekly|checkout|thankyou|thank-you))(/.*)?$',
```

Comment:
Is the $ in the weekly checkout path doing the right thing here? I think that anchors to the end, but we're then specifying that their might be some optional characters after (which can't happen if we already anchored to the end). What characters might appear after `/subscribe/weekly/checkout`? If none then we could lose the `(/.*)?`, or if it's possible we should probably lose the `$` right after checkout, e.g.

```suggestion
			'(/subscribe/weekly/checkout)(/.*)?$|(?:/(uk|us|ca|eu|nz|int))(?:/(subscribe/weekly|checkout|thankyou|thank-you))(/.*)?$',
```